modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -388,6 +393,9 @@ public Organization patchOrganization(
 			serviceBuilderOrganization.getCountryId(), organization);
 		Group group = serviceBuilderOrganization.getGroup();
 
+		serviceBuilderOrganization = _updateNestedResources(
+			organization, serviceBuilderOrganization);
+
```

Comment:
I think it's wrong, we should extract the line 401 (`_organizationService.updateOrganization(`) in the variable and put the `_updateNestedResources` result in the `toDTO` method.
Otherwise we are updating the nested resources before the "model"